<ui version="4.0" >
 <class>GuiMobInit</class>
 <widget class="QDialog" name="GuiMobInit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QTabWidget" name="tabWidget" >
        <widget class="QWidget" name="tab_eq" >
         <attribute name="title" >
          <string>Equipaggiamento</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="8" column="2" >
           <widget class="QToolButton" name="tb_feet" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="cb_ringDx" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Al collo(1):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" >
           <widget class="QLabel" name="label_20" >
            <property name="text" >
             <string>Inpugnato:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" >
           <widget class="QComboBox" name="cb_head" />
          </item>
          <item row="4" column="1" >
           <widget class="QComboBox" name="cb_neck2" />
          </item>
          <item row="10" column="2" >
           <widget class="QToolButton" name="tb_arms" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2" >
           <widget class="QToolButton" name="tb_body" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Come scudo:</string>
            </property>
            <property name="buddy" >
             <cstring>cb_zone</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QToolButton" name="tb_ringSx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QComboBox" name="cb_ringSx" />
          </item>
          <item row="5" column="0" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>Sul corpo:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" >
           <widget class="QLabel" name="label_19" >
            <property name="text" >
             <string>Al polso sinistro:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Intorno al corpo:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" >
           <widget class="QLabel" name="label_14" >
            <property name="text" >
             <string>Ai piedi:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Anello sinistro:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" >
           <widget class="QComboBox" name="cb_legs" />
          </item>
          <item row="9" column="2" >
           <widget class="QToolButton" name="tb_hands" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4" >
           <widget class="QComboBox" name="cb_wield" />
          </item>
          <item row="3" column="2" >
           <widget class="QToolButton" name="tb_neck1" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2" >
           <widget class="QToolButton" name="tb_legs" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QToolButton" name="tb_light" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" >
           <widget class="QLabel" name="label_16" >
            <property name="text" >
             <string>Sulle braccia:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" >
           <widget class="QComboBox" name="cb_body" />
          </item>
          <item row="3" column="1" >
           <widget class="QComboBox" name="cb_neck1" />
          </item>
          <item row="3" column="3" >
           <widget class="QLabel" name="label_18" >
            <property name="text" >
             <string>Al polso destro:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" >
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Sulle mani:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" >
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>Sulle gambe:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3" >
           <widget class="QLabel" name="label_24" >
            <property name="text" >
             <string>All'orecchio sinistro:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4" >
           <widget class="QComboBox" name="cb_wristSx" />
          </item>
          <item row="2" column="4" >
           <widget class="QComboBox" name="cb_waist" />
          </item>
          <item row="1" column="2" >
           <widget class="QToolButton" name="tb_ringDx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3" >
           <widget class="QLabel" name="label_21" >
            <property name="text" >
             <string>Tenuto:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" >
           <widget class="QComboBox" name="cb_hands" />
          </item>
          <item row="1" column="4" >
           <widget class="QComboBox" name="cb_cloak" />
          </item>
          <item row="3" column="4" >
           <widget class="QComboBox" name="cb_wristDx" />
          </item>
          <item row="10" column="1" >
           <widget class="QComboBox" name="cb_arms" />
          </item>
          <item row="0" column="4" >
           <widget class="QComboBox" name="cb_shield" />
          </item>
          <item row="2" column="3" >
           <widget class="QLabel" name="label_17" >
            <property name="text" >
             <string>Alla vita:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" >
           <widget class="QComboBox" name="cb_feet" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Anello destro:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" >
           <widget class="QToolButton" name="tb_head" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="10" column="3" >
           <widget class="QLabel" name="label_25" >
            <property name="text" >
             <string>Sugli Occhi:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Al collo(2):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5" >
           <widget class="QToolButton" name="tb_cloak" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cb_light" />
          </item>
          <item row="6" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Sulla testa:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3" >
           <widget class="QLabel" name="label_23" >
            <property name="text" >
             <string>All'orecchio destro:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" >
           <widget class="QToolButton" name="tb_neck2" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" >
           <widget class="QLabel" name="label_22" >
            <property name="text" >
             <string>Sulla schiena:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4" >
           <widget class="QComboBox" name="cb_hold" />
          </item>
          <item row="0" column="5" >
           <widget class="QToolButton" name="tb_shield" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Come luce:</string>
            </property>
            <property name="buddy" >
             <cstring>cb_zone</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="4" >
           <widget class="QComboBox" name="cb_back" />
          </item>
          <item row="8" column="4" >
           <widget class="QComboBox" name="cb_earDx" />
          </item>
          <item row="9" column="4" >
           <widget class="QComboBox" name="cb_earSx" />
          </item>
          <item row="10" column="4" >
           <widget class="QComboBox" name="cb_eye" />
          </item>
          <item row="2" column="5" >
           <widget class="QToolButton" name="tb_waist" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5" >
           <widget class="QToolButton" name="tb_wristDx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5" >
           <widget class="QToolButton" name="tb_wristSx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="5" column="5" >
           <widget class="QToolButton" name="tb_wield" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="6" column="5" >
           <widget class="QToolButton" name="tb_hold" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="5" >
           <widget class="QToolButton" name="tb_back" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="8" column="5" >
           <widget class="QToolButton" name="tb_earDx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="9" column="5" >
           <widget class="QToolButton" name="tb_earSx" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="10" column="5" >
           <widget class="QToolButton" name="tb_eye" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_inv" >
         <attribute name="title" >
          <string>Inventario</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QListWidget" name="lw_item" />
          </item>
          <item row="0" column="1" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QToolButton" name="tb_addItem" >
              <property name="text" >
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_cloneItem" >
              <property name="text" >
               <string>:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_editItem" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_removeItem" >
              <property name="text" >
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_seg" >
         <attribute name="title" >
          <string>Seguaci</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QToolButton" name="tb_addFollowers" >
              <property name="text" >
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_cloneFollowers" >
              <property name="text" >
               <string>:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_editFollowers" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_removeFollowers" >
              <property name="text" >
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <widget class="QListWidget" name="lw_followers" />
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Zona:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_zone" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Stanza:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_room" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Limite:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_limit" >
          <property name="suffix" >
           <string/>
          </property>
          <property name="prefix" >
           <string/>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_okButton" >
       <property name="text" >
        <string>&amp;Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_reset" >
       <property name="text" >
        <string>&amp;Ripristina</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancelButton" >
       <property name="text" >
        <string>&amp;Chiudi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_okButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiMobInit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>466</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiMobInit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>652</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
